litellm/litellm_core_utils/prompt_templates/factory.py

```diff
@@ -2376,7 +2380,7 @@ def _validate_format(mime_type: str, image_format: str) -> str:
             # Use first valid extension instead of provided image_format
             return valid_extensions[0]
         else:
-            if image_format not in supported_image_formats:
+            if image_format not in supported_video_formats and image_format not in supported_image_formats:
```

Comment:
change this line to 

`if image_format not in (supported_video_formats+ supported_image_formats)`